cmd/skopeo/sync.go

```diff
@@ -186,6 +198,16 @@ func destinationReference(destination string, transport string) (types.ImageRefe
 			return nil, errors.Wrapf(err, "Error creating directory for image %s", destination)
 		}
 		imageTransport = directory.Transport
+	case oci.Transport.Name():
+		dir := path.Dir(destination)
+		if err := os.MkdirAll(destination, 0755); err != nil {
+			return nil, errors.Wrapf(err, "Error creating directory:%s for image %s", dir, destination)
+		}
+		if ref, err := reference.ParseNormalizedNamed(destination); err != nil {
+			tagref, _ := reference.TagNameOnly(ref).(reference.Tagged)
+			destination = fmt.Sprintf("%s:%s", reference.Path(ref), tagref)
```

Comment:
Isn’t this building $top-dir/$repo directories, one for each repository?

Why build an entirely new directory structure on top of OCI, when we can just build one large OCI repository?